runtime/server_http_request.go

```diff
@@ -30,31 +30,24 @@ import (
 	"time"
 
 	"github.com/julienschmidt/httprouter"
-	"github.com/uber-go/tally"
 	"go.uber.org/zap"
 )
 
 // ServerHTTPRequest struct manages request
 type ServerHTTPRequest struct {
 	httpRequest *http.Request
-	res         *ServerHTTPResponse
-	gateway     *Gateway
+	Res         *ServerHTTPResponse
 	started     bool
-	startTime   time.Time
```

Comment:
please keep these fields private.

Reply:
Sure.  I can keep them private.  I made them public since they were accessed by other runtime classes, but they are all under the same package, so should be fine to make them private.